metrics/testdata/prometheus_metrics

```diff
@@ -341,6 +341,9 @@ container_network_udp_usage_total{container_env_foo_env="prod",container_label_f
 container_network_udp_usage_total{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",udp_state="listen",zone_name="hello"} 0 1395066363000
 container_network_udp_usage_total{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",udp_state="rxqueued",zone_name="hello"} 0 1395066363000
 container_network_udp_usage_total{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",udp_state="txqueued",zone_name="hello"} 0 1395066363000
+# HELP container_perf_event_errors Perf event errors.
+# TYPE container_perf_event_errors gauge
+container_perf_event_errors{action="some_action",container_env_foo_env="prod",container_label_foo_label="bar",event="non_existing_event",id="testcontainer",image="test",name="testcontaineralias",zone_name="hello"} -4 1395066363000
```

Comment:
In really collection if there is an error we will have multiple errors for many CPUs or many PMUs and sockets in this format of metric we will have many metrics with the same labels and the same value. Have you checked how it works with Prometheus?
Please add more complicated case to unit tests.